test/unsplash-test.js

```diff
@@ -329,7 +329,7 @@ describe("Unsplash", () => {
     describe("getRandomPhoto", () => {
       it("should make a GET request to /photos/random", () => {
         let spy = spyOn(unsplash, "request");
-        unsplash.photos.getRandomPhoto();
+        unsplash.photos.getRandomPhoto(undefined,undefined,undefined,undefined,undefined,undefined, 123);
```

Comment:
It's better to pass `null` instead of `undefined` here.

`null` implies that no value was passed whereas `undefined` means something has not been declared.

Previously the test checked for `undefined` since no value was being passed at all.  However, now we're explicitly passing falsy values in order to set the cache buster.